.golangci.bck.yml

```diff
@@ -0,0 +1,117 @@
+linters:
```

Comment:
had to migrate to golangcilint to work ...

I just ran `golangci-lint migrate`